common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -319,9 +339,16 @@ def get_pending_reviews(self, user_id):
         url = self.pending_reviews_url
         student_uid = self.generate_student_uid(user_id)
         payload = {'student_uid': student_uid, 'nested': True}
-        response = requests.get(url, headers=self.request_headers, data=payload)
-        if response.content == 'No Student Found':
+        content = 'No Student Found'
+        try:
+            response = requests.get(url, headers=self.request_headers, data=payload)
+            content = response.content
+        except requests.ConnectionError:
             pending_reviews_user = []
+            log.warning("Could not get pending reviews for user identified by user_id {user_id}.".format(user_id=user_id))
```

Comment:
@macornwell This line is too long; please add some line breaks to make it shorter.